.github/renovate.json

```
@@ -5,11 +5,11 @@
   "packageRules": [
     {
       "matchPackageNames": ["@types/node"],
-      "allowedVersions": "<13"
+      "allowedVersions": "<14"
```

This must be "less than 15".

```suggestion
      "allowedVersions": "<15"
```